From 831b57ff2167e1192964c190271767e469dac48a Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Tue, 3 Oct 2017 05:01:21 +1300
Subject: Fix Test::Builder warning

Ideally upstream should just stop bundling Test::Builder, but this
is the smallest change that avoids the issue

Bug: https://rt.cpan.org/Ticket/Display.html?id=111009
---
 inc/Test/Builder.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/inc/Test/Builder.pm b/inc/Test/Builder.pm
index 795361f..6b0c8c3 100644
--- a/inc/Test/Builder.pm
+++ b/inc/Test/Builder.pm
@@ -589,7 +589,7 @@ sub _is_dualvar {
 
     no warnings 'numeric';
     my $numval = $val + 0;
-    return $numval != 0 and $numval ne $val ? 1 : 0;
+    return ($numval != 0 and $numval ne $val ? 1 : 0);
 }
 
 #line 876
-- 
2.14.1

